examples/jsm/nodes/shadernode/ShaderNodeBaseElements.js

```diff
@@ -297,3 +297,11 @@ export const luminance = ( color, luma = lumaCoeffs ) => dot( color, luma );
 export const difference = ( a, b ) => abs( sub( a, b ) );
 export const dotNV = clamp( dot( transformedNormalView, positionViewDirection ) );
 export const TBNViewMatrix = mat3( tangentView, bitangentView, normalView );
+
+export const posterize = ( value, steps ) => {
+
+	const reciprocalSteps = div( 1, steps );
+
+	return mul( floor( div( value, reciprocalSteps ) ), reciprocalSteps );
```

Comment:
Isn't division of the value by reciprocal steps just multiplcation of value with steps?

Reply:
I mean `value / (1 / steps) = value * steps`.

Reply:
It works, thanks :)